commit af007961e2fd0608aff556dfe4bf9e537e9154c3
Author: Dmitry Grinberg <dmitrygr@gmail.com>
Date:   Mon Dec 26 02:26:57 2022 -0600

    use WinDrawChars in place of WinPaintChars
    
    not sure why paint was used there, no changes noted

diff --git a/Src/PkmnType.c b/Src/PkmnType.c
index 314bbf9..81e3cf6 100644
--- a/Src/PkmnType.c
+++ b/Src/PkmnType.c
@@ -106,16 +106,16 @@ static void DrawEffectiveness(UInt16 selectedPkmnID, UInt8 x, UInt8 y, UInt8 typ
 	}
 
     x += 35;
-	WinPaintChars("x ", 2, x, y);
+	WinDrawChars("x ", 2, x, y);
 	
 	x += 7;
 	if (effectiveness == HALF_DAMAGE)
 	{
-		WinPaintChars("0.5", 3, x, y);
+		WinDrawChars("0.5", 3, x, y);
 	}
 	else if (effectiveness == QUARTER_DAMAGE)
 	{
-		WinPaintChars("0.25", 4, x, y);
+		WinDrawChars("0.25", 4, x, y);
 	}
 	else 
 	{
@@ -123,7 +123,7 @@ static void DrawEffectiveness(UInt16 selectedPkmnID, UInt8 x, UInt8 y, UInt8 typ
 		ErrFatalDisplayIf ((UInt32)str == 0, "Out of memory");
 		MemSet(str, sizeof(Char[4]), 0);
 		StrIToA(str, effectiveness/100);
-    	WinPaintChars(str, StrLen(str), x, y);
+    	WinDrawChars(str, StrLen(str), x, y);
     
 		MemPtrFree(str);
 	}
